cli/utils/calculate-rewards.ts

```diff
@@ -1,6 +1,27 @@
 import { BigNumber } from 'bignumber.js'
 import { EventLog } from 'web3-core'
 
+const tierRewards: {[keys: number]: number} = {
+  20e18: 1e18,
+  100e18: 5e18,
+  500e18: 10e18 
+}
+const tiers = Object.keys(tierRewards).map(s => new BigNumber(parseInt(s)))
```

Comment:
the parseInt is superflous, right?

Reply:
not bc of Object.keys => array of strings